lib/cli/kit/executor.rb

```diff
@@ -16,8 +16,14 @@ def call(command, command_name, args)
             begin
               command.call(args, command_name)
             rescue => e
-              $stderr.puts "This command ran with ID: #{id}"
-              $stderr.puts "Please include this information in any issues/report along with relevant logs"
+              begin
+                $stderr.puts "This command ran with ID: #{id}"
+                $stderr.puts "Please include this information in any issues/report along with relevant logs"
+              rescue
```

Comment:
This masks things like `SyntaxError` and `NameError` too. Maybe rescue `SystemCallError`?